tests/model_explainability/guardrails/conftest.py

```diff
@@ -49,8 +47,8 @@ def prompt_injection_detector_isvc(
         deployment_mode=KServeDeploymentType.RAW_DEPLOYMENT,
         model_format="guardrails-detector-huggingface",
         runtime=huggingface_sr.name,
-        storage_key=minio_data_connection.name,
-        storage_path="deberta-v3-base-prompt-injection-v2",
+        storage_uri="oci://quay.io/trustyai_testing/detectors/deberta-v3-base-prompt-injection-v2"
+        "@sha256:8737d6c7c09edf4c16dc87426624fd8ed7d118a12527a36b670be60f089da215",
```

Comment:
Instead of hardcoding this storge_uri can we call this from utilities/constants.py.